verl/workers/rollout/sglang_rollout/sglang_rollout.py

```diff
@@ -767,10 +809,32 @@ async def _async_rollout_a_request(
                             _req.state = AsyncRolloutRequestStateEnum.COMPLETED
                             break
                     else:
-                        _req.add_assistant_message(self.tokenizer, content)
+                        _req.add_assistant_message(
+                            self.tokenizer,
+                            content,
+                        )
+                        if _req.interaction_kwargs and user_turns < self.config.multi_turn.max_user_turns and current_turns < self.config.multi_turn.max_assistant_turns:
+                            _req.state = AsyncRolloutRequestStateEnum.INTERACTING
+                        else:
+                            break
+            elif _req.state == AsyncRolloutRequestStateEnum.INTERACTING:
+                user_turns += 1
+                messages = [{"role": x.role, "content": x.content} for x in _req.messages]
+                should_terminate_sequence, content, reward, metrics = await self.interaction.generate_response(_req.request_id, messages, **_req.interaction_kwargs)
```

Comment:
Do we support choose different interaction based on assistant response? Or we just use one interaction, and dispatch in it?